less/index.less

```diff
@@ -53,7 +55,179 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html{
+	font-size: 62.5%;
+	width: 100%;
+	@media( max-width: 500px ) {
+		border: 10px solid red;
+		margin: 0 auto;
+	}
+}
+
+// html, body {
+// 	height: 100%;
+	
+// }
+
+@secondary-color: midnightblue;
+@main-color: royalblue;
+@tertiary-color: cornflowerblue;
+@accent-color: lavender;
+
+// MIXINS
+.border(){
+	border: 20px double @accent-color;
+	padding: 2%;
+  }
+
+.display(){
+	display: flex;
+	flex-direction: row;
+}  
+
+.padding(){
+	width: 48%;
+	padding: 2%;
+}
+
+
+.mediaPadding(){
+	width: auto;
+	padding: 2%;
+}
+
+
+header{
+	display: flex;
+	flex-direction: row;
+	
+}
+.navigation{
+	display:flex;
+	justify-content: space-around;	
+	width:90%;
+
+	a{
+		color: @accent-color;
+		padding: 1%;
+	}
+}
+
+body{
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	font-size: 2rem;
+	padding: 2%;
+	background-color: @main-color;
+	color: @accent-color;
+	@media( max-width: 500px ) {
```

Comment:
You incorporated media query responsiveness.  This demonstrates that you understand how to use it, so good job!